.env

```diff
@@ -0,0 +1 @@
+DOCKER_TAG=latest
```

Comment:
Since this is back now, can you remove the `latest` default values you adding in both docker-compose files?